generator/src/bin/generator.rs

```diff
@@ -2,5 +2,17 @@ use generator::write_source_code;
 use std::path::Path;
 
 fn main() {
-    write_source_code(Path::new("Vulkan-Headers/registry/vk.xml"));
+    let cwd = std::env::current_dir().unwrap();
+    let cwd_str = cwd.as_os_str().to_str().unwrap();
```

Comment:
Why are you converting it to a str instead of using https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.ends_with ?